MaterialDesignThemes.Wpf/SmartHint.cs

```diff
@@ -17,15 +16,13 @@ namespace MaterialDesignThemes.Wpf
     /// <para/>
     /// To set a target control you should set the HintProxy property. Use the <see cref="HintProxyFabricConverter.Instance"/> converter which converts a control into the IHintProxy interface.
     /// </summary>
-    [TemplateVisualState(GroupName = ContentStatesGroupName, Name = ContentEmptyName)]
-    [TemplateVisualState(GroupName = ContentStatesGroupName, Name = ContentNotEmptyName)]
+    [TemplateVisualState(GroupName = ContentStatesGroupName, Name = HintRestingPositionName)]
+    [TemplateVisualState(GroupName = ContentStatesGroupName, Name = HintFloatingPositionName)]
     public class SmartHint : Control
     {        
         public const string ContentStatesGroupName = "ContentStates";
-        public const string ContentEmptyName = "ContentEmpty";
-        public const string ContentNotEmptyName = "ContentNotEmpty";
-
-        private ContentControl _floatingHintPart = null;
+        public const string HintRestingPositionName = "HintRestingPosition";
```

Comment:
There is a small issue with renaming this.  They are part of the public API, and we are following semver.  Admittedly, I'd be a but surprised if people were accessing this in their code, but i'd like to stick to semver wherever possible and don't want to tick a major version.

Can we please rename them back.  Which might also mean that ``` IsHintInFloatingPosition ``` might need to be named in according with the original state names.